doc/modules/grid_search.rst

```diff
@@ -1,10 +1,10 @@
-.. currentmodule:: sklearn.grid_search
+.. currentmodule:: sklearn.model_selection
 
 .. _grid_search:
 
-===============================================
-Grid Search: Searching for estimator parameters
-===============================================
+==========================================
+Search: Searching for estimator parameters
```

Comment:
Maybe "Adjusting hyperparameters for estimators"
or just  "Searching for estimator parameters" or "Searching for estimator hyper-parameters". I feel "adjusting" is better then "Searching".